common/compositor/gl/glprogram.cpp

```diff
@@ -231,6 +237,9 @@ void GLProgram::UseProgram(const RenderState &state, GLuint viewport_width,
               (state.width_) / (float)viewport_width,
               (state.height_) / (float)viewport_height);
 
+  glUniformMatrix4fv(ctm_loc_, 1, GL_FALSE, ctm_);
```

Comment:
Do we need to set this always ? Can we have a flag to track any change to these values and if so do this.